package.json

```diff
@@ -72,6 +70,7 @@
     "strip-ansi": "^6.0.1",
     "swc-node": "^1.0.0",
     "ts-jest": "^29.2.4",
+    "tstyche": "^4.0.0-beta.6",
```

Comment:
Only the latest version has `.toBeCallableWith()` matcher. It is still in beta. Stable version will be available in month. CLI usage will not break any more.